apps/docs/src/app/platform/component-docs/platform-approval-flow/platform-approval-flow-examples/platform-approval-flow-example.component.html

```diff
@@ -1,7 +1,10 @@
 <fdp-approval-flow
     title="Basic Approval Flow Demo"
-    [dataSource]="dataSource"
+    [approvalProcessData]="approvalProcess"
     [userDetailsTemplate]="userDetailsTemplate"
+    [usersDataSource]="usersDataSource"
```

Comment:
User not users, userDataSource  - Data Source for a User Type

Reply:
done